frontend/src/metabase/admin/datamodel/metadata/components/FieldGeneralSettings/FieldGeneralSettings.tsx

```diff
@@ -89,7 +89,7 @@ const FieldHeaderSection = ({
   onUpdateField,
 }: FieldHeaderSectionProps) => {
   const handleChangeName = useCallback(
-    (event: ChangeEvent<HTMLInputElement>) => {
```

Comment:
why do you need to change this? To me `Event<Element>` is more obvious and "correct" comparing to `{target: ..}`